pkgs/tools/misc/atuin/default.nix

```diff
@@ -37,6 +37,16 @@ rustPlatform.buildRustPackage rec {
       --zsh <($out/bin/atuin gen-completions -s zsh)
   '';
 
+  nativeCheckInputs = lib.optionals (lib.meta.availableOn stdenv.buildPlatform xvfb-run) [
+    xvfb-run
```

Comment:
Undeclared.

Reply:
Ah, wonderful, #235406 has made this unnecessary!